### PR TITLE
2719 - Material UI - Icons

### PR DESCRIPTION
**Pull request by evasileva-epam**

### CODE REVIEW

File: jdi-light-material-ui-tests/src/test/java/io/github/epam/material/tests/datadisplay/IconsTests.java [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    public void simpleIconsTest() {
+        simpleIconsPage.open();
+
+        iconsFrame.filled1.is().displayed();
+        iconsFrame.filled2.is().displayed();
+        iconsFrame.oulined1.is().displayed();
+    }
+
+    @Test
+    public void colorIconsTest() {
```
Comment (kborovinsky25) [inline jdi-light-material-ui-tests/src/test/java/io/github/epam/material/tests/datadisplay/IconsTests.java:20]: Can we check the color of the icon in this case?

File: jdi-light-material-ui-tests/src/test/java/io/github/epam/material/tests/datadisplay/IconsTests.java [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+        colorIconsPage.open();
+
+        iconsFrame.svg1.is().displayed();
+        iconsFrame.svg2.is().displayed();
+    }
+
+    @Test
+    public void sizeIconsPage(){
```
Comment (kborovinsky25) [inline jdi-light-material-ui-tests/src/test/java/io/github/epam/material/tests/datadisplay/IconsTests.java:28]: Can we check the size of the icon in this case?

File: jdi-light-material-ui-tests/src/main/java/io/github/com/StaticSite.java [hunk 3]
```diff
@@ -52,6 +53,20 @@
     @Url("/material-ui-surfaces-accordion--disabled")
     public static WebPage surfaceAccordionDisabledPage;
 
+    // data display
+    //icons
+    @Frame("storybook-preview-iframe")
+    public static IconsFrame iconsFrame;
+
+    @Url("/material-ui-icons--simple-icons")
```
Comment (LunAndrey) [inline jdi-light-material-ui-tests/src/main/java/io/github/com/StaticSite.java:61]: need to change url to
material-ui-data-display-icons--simple-icons

File: jdi-light-material-ui-tests/src/main/java/io/github/com/pages/datadisplay/IconsFrame.java [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+    @UI("//div[contains(@class, 'MuiGrid-grid-xs-8')][1]/*[name()='svg'][1]")
+    public Icons filled1;
+
+    @UI("//div[contains(@class, 'MuiGrid-grid-xs-8')][1]/*[name()='svg'][2]")
+    public Icons filled2;
+
+    @UI("//div[contains(@class, 'MuiGrid-grid-xs-8')][2]/*[name()='svg'][1]")
+    public Icons oulined1;
+
+    @UI("//*[name()='svg'][1]")
```
Comment (LunAndrey) [inline jdi-light-material-ui-tests/src/main/java/io/github/com/pages/datadisplay/IconsFrame.java:17]: this locator incorrect (many matches on page)
need to replace on .MuiSvgIcon-root

File: jdi-light-material-ui-tests/src/main/java/io/github/com/pages/datadisplay/IconsFrame.java [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+    public Icons svg1;
+
+    @UI("//*[name()='svg'][2]")
```
Comment (LunAndrey) [inline jdi-light-material-ui-tests/src/main/java/io/github/com/pages/datadisplay/IconsFrame.java:20]: this locator incorrect (many matches on page)
for other check (blue icon) need to use this locator .MuiSvgIcon-colorPrimary

File: jdi-light-material-ui-tests/src/test/java/io/github/epam/material/tests/datadisplay/IconsTests.java (hunk 1 above)
Comment (evasileva-epam) [reply jdi-light-material-ui-tests/src/test/java/io/github/epam/material/tests/datadisplay/IconsTests.java:20]: I added check for color

File: jdi-light-material-ui-tests/src/test/java/io/github/epam/material/tests/datadisplay/IconsTests.java (hunk 2 above)
Comment (evasileva-epam) [reply jdi-light-material-ui-tests/src/test/java/io/github/epam/material/tests/datadisplay/IconsTests.java:29]: I added check for size

**Pull request merged**